domain/src/main/kotlin/no/nav/su/se/bakover/domain/Sak.kt

```diff
@@ -407,6 +415,10 @@ data class Sak(
         startDato: LocalDate,
         clock: Clock,
     ): Either<KunneIkkeOppretteEllerOppdatereRegulering, Regulering.OpprettetRegulering> {
+        // har ikke en reguleringssjekk her, da vi gjør det i filterIsInstance()
+        if (harÅpenSøknadsbehandling() || hentÅpneRevurderinger().isRight()) {
+            return KunneIkkeOppretteEllerOppdatereRegulering.HarÅpenBehandling.left()
+        }
```

Comment:
Hva med å lage en fuksjon som kan brukes for alle tilfeller? Da slipper vi sånne quirks som "dette behøver vi ikke fordi vi filtrerer senere" o.l - den vil alltid gi oss det svaret vi er ute etter:
`fun kanOppretteBehandling():Boolean {
   return !harÅpenSøknadsbehandling() && !harÅpenRevurdering() && !harÅpenRegulering
}`